spec/helpers/claims/show_helper_spec.rb

```diff
@@ -38,4 +38,76 @@
       end
     end
   end
+
+  describe "#base_and_uplift_award_amounts(claim)" do
+    context "with a StudentLoans policy" do
+      let(:policy) { StudentLoans }
+
+      it "returns nil" do
+        expect(helper.base_and_uplift_award_amounts(claim, nil)).to be_nil
+      end
+    end
+
+    context "with a EarlyCareerPayments policy" do
+      let(:policy) { EarlyCareerPayments }
+
+      describe "Chemistry/Physics/Foreign Languages" do
```

Comment:
Again, this kind of thing should be context.

Reply:
Is it ok to have context, context, context? I thought the linters got fussy and liked 2 levels of describes, then multiple contexts if more than 3 levels deep.
Maybe thats rubocop which we are overriding with `standard` which I have to say personally I am not a fan of